test/multiverse/suites/rails/activejob_test.rb

```diff
@@ -131,6 +131,16 @@ def test_record_perform_metrics_in_web
       assert_metrics_recorded("#{PERFORM_PREFIX}/default")
     end
 
+    def test_record_perform_all_later_metrics_in_web
+      skip if Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new('7.1.0')
+
+      in_web_transaction do
+        ActiveJob.perform_all_later(MyJob.new)
```

Comment:
[The docs](https://edgeapi.rubyonrails.org/classes/ActiveJob.html#method-c-perform_all_later) for this method make it sound like the method is intended for multiple jobs. Would you mind adding a test that looks at how the agent behaves when multiple jobs are given to `perform_all_later`?

Reply:
Great call out! Updated the test to include multiple jobs.